.lando.yml

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
can we move this file to the `dockerfiles` dir and pass the path as an argument to lando, similar to  `docker-compose` and `docker` commands behave?

Reply:
Discussed on Discord, leave it in the root for now to save typing.